test/functional/autocmd/autocmd_oldtest_spec.lua

```diff
@@ -0,0 +1,86 @@
+local helpers = require('test.functional.helpers')(after_each)
+
+local clear = helpers.clear
+local eq = helpers.eq
+local meths = helpers.meths
+local funcs = helpers.funcs
+
+local exec = function(str)
+  meths.exec(str, false)
+end
+
+describe('oldtests', function()
```

Comment:
A bit weird of a name. If the point is that it is a port of a vim oldtest, this file should be called `legacy/autocmd_spec.lua` or something.